proto_task_queue/integration_test.py

```diff
@@ -49,6 +49,8 @@
 class ProtoTaskQueueTest(absltest.TestCase):
 
   def setUp(self):
+    super(ProtoTaskQueueTest, self).setUp()
```

Comment:
python 3 has some super magic so you can only do `super().setUp()`

Reply:
I forgot about that, thanks!